packages/data-context/src/sources/BrowserDataSource.ts

```diff
@@ -33,6 +42,41 @@ export interface BrowserApiShape {
 export class BrowserDataSource {
   constructor (private ctx: DataContext) {}
 
+  /**
+   * Gets the browsers from the machine and the project config
+   */
+  allBrowsers () {
```

Comment:
I think we can move this to use the `async/await` style - I know you are just matching the code style, but in general I think we are moving away from big nested `then` chains in favor of the more modern async/await. Thoughts?

Reply:
Fine with me. As you guessed, I was just matching style